packages/examples/lumberjack-app/src/app/app.component.spec.ts

```diff
@@ -3,13 +3,11 @@ import { TestBed } from '@angular/core/testing';
 import { LumberjackModule } from '@ngworker/lumberjack';
 
 import { AppComponent } from './app.component';
-import { NxWelcomeComponent } from './nx-welcome.component';
 
 describe('AppComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [AppComponent, NxWelcomeComponent],
-      imports: [LumberjackModule.forRoot()],
+      imports: [AppComponent, LumberjackModule.forRoot()],
```

Comment:
issue: It's not neccessary to import standalone components in component tests
```suggestion
      imports: [LumberjackModule.forRoot()],
```